WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -230,9 +239,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
         mLayoutReply = (ViewGroup) view.findViewById(R.id.layout_comment_box);
         mEditReply = (SuggestionAutoCompleteText) mLayoutReply.findViewById(R.id.edit_comment);
-        mEditReply.getAutoSaveTextHelper().setUniqueId(String.format("%s%d%d",
-                AccountHelper.getCurrentUsernameForBlog(WordPress.getCurrentBlog()),
-                getRemoteBlogId(), getCommentId()));
+        mEditReply.getAutoSaveTextHelper().setUniqueId(String.format(Locale.US, "%d%d", getRemoteBlogId(),
```

Comment:
This unique id is used to retrieve typed text that wasn't sent yet. Having a username here was in case of user switching accounts (logout/login with another account), but it's an extreme case and it's not really an issue (even with another account, user could need that saved text).